clio/tpl/ui-project/angular.json

```diff
@@ -74,13 +64,39 @@
 					},
 					"defaultConfiguration": "production"
 				},
+				"serve": {
+					"builder": "ngx-build-plus:dev-server",
+					"configurations": {
+						"production": {
+							"browserTarget": "<%projectName%>:build:production",
+							"extraWebpackConfig": "webpack.prod.config.js"
```

Comment:
@besgit clio does not contain template for 
webpack.prod.config.js, where is this coming form ?